includes/functions/llms.functions.templates.loop.php

```diff
@@ -1,47 +1,77 @@
 <?php
 /**
- * Template functions for the student dashboard
+ * Template functions for loops (catalogs)
  *
  * @package LifterLMS/Functions
  *
  * @since 1.0.0
- * @version 4.0.0
+ * @version [version]
  */
 
 defined( 'ABSPATH' ) || exit;
 
-/**
- * If content added to the course/membership catalog page, output it as the archive description before the loop
- *
- * @return   void
- * @since    3.16.10
- * @version  3.19.0
- */
 if ( ! function_exists( 'lifterlms_archive_description' ) ) {
+	/**
+	 * Output the archived description for LifterLMS catalogs pages and post type / tax archives.
+	 *
+	 * @since 3.16.10
+	 * @since 3.19.0 Unknown.
+	 * @since [version] Moved logic to `lifterlms_get_archive_description()` so to function can be called without outputting the content.
```

Comment:
```suggestion
	 * @since [version] Moved logic to `lifterlms_get_archive_description()` so the function can be called without outputting the content.
```